docs/quickstart/invoice_model.mdx

```diff
@@ -48,28 +60,33 @@ Before we enter Cradl to start the training, we make sure that we have data to t
 </TabItem>
 </Tabs>
 
-## 2. Configure your model
-We are now ready to enter Cradl and put our data to use.
-1. In Cradl, go to [**Models**](https://app.cradl.ai/models) > [**New model**](https://app.cradl.ai/models/new).
-2. Give your model a name, and add an optional description.
-3. Choose the invoice template and press **Create model**.
-4. In the *Model overview* section press **Go to training** and then **New training**.
+:::caution
+Make sure that the labels in your ground truth files matches exactly the names of the fields you defined in your model.
+:::
+
+In the example above, notice that the due date is written as an ISO formatted date __"2019-02-26"__ even though on the document it is written as __"26/02/2019"__.
+This is perfectly fine as long as you do this consistently for all dates in your ground truth. This can also be useful for other fields which require _normalization_,
+such as amounts where you can 
+write e.g. __"1500.36"__ in the ground truth even if it says e.g. __"$1.500,36"__ on the document.
```

Comment:
Consistent formatting is not only perfectly fine but also required, should we mention this here as well? With the current wording it sounds a bit optional to me